src/components/TextField/TextField.tsx

```diff
@@ -200,42 +196,44 @@ export const TextField = React.forwardRef<HTMLInputElement, TextFieldProps>(
 
     return (
       <div className={clsx(classes.root, className)}>
-        <label
-          aria-hidden="true"
-          className={clsx(
-            classes.label,
-            color === 'inverse' && classes.labelInverse,
-            !label && ariaLabel && classes.srOnly
-          )}
-          htmlFor={uniqueId}
-        >
-          {label || ariaLabel}
-          {!!Icon && tooltipMessage && (
-            <Tooltip title={tooltipMessage}>
-              <span className={classes.tooltipContainer}>
-                <Icon
-                  className={clsx(
-                    classes.labelIcon,
-                    color === 'inverse' && classes.labelIconInverse
-                  )}
-                  width={16}
-                  height={16}
-                  role="img"
-                  aria-hidden
-                />
-              </span>
-            </Tooltip>
-          )}
-          {secondaryLabel && (
-            <span className={classes.labelSecondary}>{secondaryLabel}</span>
-          )}
-        </label>
+        {label && (
+          <label
+            aria-hidden="true"
```

Comment:
I'm curious if we can/should delete this as well?

Reply:
I don't think we need it anymore.